store/localstore/local_region.go

```diff
@@ -505,7 +689,7 @@ func (rs *localRegion) getRowsFromIndexReq(ctx *selectContext) error {
 		if limit == 0 {
 			break
 		}
-		count, err := rs.getIndexRowFromRange(ctx, ran, desc, limit)
+		count, err := rs.getIndexRowFromRange(ctx, ran, ctx.descScan, limit)
```

Comment:
function "getRowsFromRange" also needs this var. So it's better to calculate it outside.